src/ekka.appup.src

```diff
@@ -1,20 +1,13 @@
 %% -*-: erlang -*-
-
-{"0.7.6",
-  [
-    {"0.7.5", [
-      {load_module, ekka_dist, brutal_purge, soft_purge, []}
-    ]},
-    {"0.7.4", [
-      {load_module, ekka_locker, brutal_purge, soft_purge, []}
-    ]}
-  ],
-  [
-    {"0.7.5", [
-      {load_module, ekka_dist, brutal_purge, soft_purge, []}
-    ]},
-    {"0.7.4", [
-      {load_module, ekka_locker, brutal_purge, soft_purge, []}
-    ]}
-  ]
+{"0.7.7",
```

Comment:
Unsure about 0.7.7 is a preferred vsn tag for emqx 4.2.x to support hot upgrading.

Reply:
It’s ok